ott/geometry/pointcloud.py

```diff
@@ -533,8 +533,8 @@ def finalize(i: int):
 
   def barycenter(self, weights: jnp.ndarray) -> jnp.ndarray:
     """Compute barycenter of points in self.x using weights, valid for p=2.0."""
```

Comment:
The docstring still says `valid for p=2.0`.

Reply:
thanks Michal for the catch!